app/views/spree/admin/orders/edit.html.haml

```diff
@@ -9,6 +9,7 @@
     %li= button_link_to t(:back_to_orders_list), admin_orders_path, :icon => 'icon-arrow-left'
 
 = render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Order Details' }
+= render :partial => 'spree/admin/shared/order_page_title', :locals => { :current => 'Order Details' }
```

Comment:
any reason to call title after tabs? It's better keep consistency with other partials and call page_title before order tabs.

Reply:
No reason, I have switched those two, consistency is indeed good.